src/Handler.php

```diff
@@ -12,24 +12,85 @@ class Handler
     protected $client;
 
     /**
+     * The previously registered error handler as returned by the interpreter.
+     *
+     * @var callable|null
+     */
+    protected $previousErrorHandler;
+
+    /**
+     * The previously registered exception handler as returned by the interpreter.
+     *
+     * @var callable|null
+     */
+    protected $previousExceptionHandler;
+
+    /**
      * Register our exception handler.
      *
      * @param \Bugsnag\Client|string|null $client client instance or api key
+     * @param bool $callPrevious whether or not to call the previous handlers
      *
      * @return static
      */
-    public static function register($client = null)
+    public static function register($client = null, $callPrevious = true)
```

Comment:
> I'm generally not the biggest fan of boolean flags either, but in this case I think removing it would either (a) add a lot of duplicated code or (b) add unnecessary complexity to the existing code.

I'm only suggesting we don't have it in the register function. The others are fine. :)